.github/workflows/release.yml

```diff
@@ -0,0 +1,127 @@
+on:
```

Comment:
It seems this will do the same as the build workflow, no? (it already takes care of publishing the artifacts to GH releases)